### ISSUE TITLE
Training data "T" condition

### ISSUE THREAD

**Issue by RickyYXY** [cvlab-columbia/zero123#93]
I find that in your evaluation code, you add T as a condition by
`T = T[None, None, :].repeat(n_samples, 1, 1).to(c.device)`
`c = torch.cat([c, T], dim=-1)`
But in your training code, I can't find where you add the T condition. 
In zero123/ldm/models/diffusion/ddpm.py, function get_input in class LatentDiffusion, 
the variant T has never been used since
`T = T[:bs].to(self.device)`,
which means the training cannot get any pose condition at all.
Is this a bug? Do you forget to add T condition here?
Thks
